server/machines/machines.controller.go

```diff
@@ -0,0 +1,34 @@
+package machines
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+type HttpHandlerFunc http.HandlerFunc
+
+func HttpHandler(ms *MachinesService) HttpHandlerFunc {
+	return func(rw http.ResponseWriter, r *http.Request) {
+		if r.Method == "GET" {
+			handleListMachines(rw, ms)
+		} else {
+			rw.WriteHeader(http.StatusMethodNotAllowed)
+		}
+	}
+}
+
+func handleListMachines(rw http.ResponseWriter, ms MachinesService) {
+	rw.Header().Set("content-type", "application/json")
```

Comment:
`git cherry-pick e0ef103bc97fdbceb6affc6e1b3fda94114bccc0` and use `Respond` function